tezedge_state/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "tezedge-state"
+version = "0.1.0"
+authors = ["Zura Benashvili <zura.benashvili@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+getset = "0.1"
+bytes = "1"
+slab = "0.4.3"
+rand = "0.7.3"
+failure = "0.1"
+futures = "0.3"
+hex = "0.4"
+riker = "0.4"
```

Comment:
is riker used for anything in this implementation?

Reply:
No its not. I need to clear dependencies in other places too.

Reply:
removed unused dependencies from tezedge-state.